src/EmojiButton.svelte

```diff
@@ -0,0 +1,10 @@
+<script>
+  export let label = "";
+  export let symbol;
+</script>
+
+<button on:click>
+  <span role="img" aria-label={label} aria-hidden={label !== ""}>
```

Comment:
I set the `label` default to `""` so instead of `label ? false : true`, you can just do `label !== ""`